logic/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "main": "../dist/logic/index.js",
   "module": "../dist/logic/index.esm.js",
-  "types": "../dist/logic-index.d.ts"
+  "types": "../dist/logic-index.d.ts",
+  "sideEffects": false
```

Comment:
Is this a left over artifact from our experiment with pkgu, or do we need it on this branch?

Reply:
Removed it and the logic bundle size is still at 7kb and the DCR bundle didn't increase in size, so removing.